src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.jsx

```diff
@@ -95,6 +96,9 @@ class OrganizationStats extends React.Component {
           )}
         </div>
         <div>
+          <Alert type="info" icon="icon-circle-info">
+            {t('Transactions and attachments are not present in the graph currently.')}
```

Comment:
Is this the phrasing we want to use?

Should we make transactions a link?

> Transactions and attachments are not yet included in the graph

Feels a little more on par with our tone.

Reply:
👍